src/plugins/auto/src/plugin.cpp

```diff
@@ -98,6 +98,16 @@ std::shared_ptr<ov::ICompiledModel> Plugin::import_model(std::istream& model,
     OPENVINO_NOT_IMPLEMENTED;
 }
 
+bool Plugin::is_meta_device(const std::string& priorities) const {
+    std::vector<std::string> candidate_devices = m_plugin_config.parse_priorities_devices(priorities);
+    for (const auto& device : candidate_devices) {
+        if (device.find("AUTO") == 0 || device.find("MULTI") == 0) {
```

Comment:
Why are only AUTO and MULTI checked here, but not HETERO and BATCH?

Reply:
https://github.com/openvinotoolkit/openvino/pull/14739

Reply:
HETERO and BATCH are allowed, they do not cause the recursive calling.